ee/app/livechat-enterprise/server/settings.js

```diff
@@ -53,24 +53,30 @@ export const createSettings = () => {
 		],
 	});
 
-	settings.add('Livechat_auto_close_abandoned_rooms', false, {
-		type: 'boolean',
+	settings.add('Livechat_abandoned_rooms_action', 'none', {
```

Comment:
Where is the migration that:
- Migrate the old setting value to the new setting;
- Remove the old setting;

Reply:
@renatobecker I've added the new migrations, could you please review it

Reply:
Your migration is not working since you didn't import it on `./index.js` file.